### DISCUSSION TITLE
EASY Request: Default Buttons/Icons to “No Action”

### DISCUSSION THREAD

**Discussion by gmoney63** [Clooos/Bubble-Card#831] [Feature requests]
PLEASE set the default button action to “No Action” so that I don’t have to go through every single button/icon tap action. This way, **_I_** can choose what happens with actions instead of of spending so much time modifying each and every one of them

**Comment by Clooos** [comment]
Hi, this is not that easy, because if I change that, all the users that are using the current default will have to change it for all there cards, like me and I know that I'm not the only one.

I don't see an easy way to do that 😕

**Comment by lukaszwawrzyk** [comment]
Yes, and people probably usually want the default action to be there, like when you tap a button, it should toggle the device or show more info or whatever. Someone might come and complain that he has to configure all buttons, instead of having sensible default :P So it is a hard pick.
